python/src/nanoarrow/_lib.pyx

```diff
@@ -858,6 +858,24 @@ cdef class CSchemaView:
     def storage_type_id(self):
         return self._schema_view.storage_type
 
+    @property
+    def buffer_format(self):
+        if not self.extension_name and self._schema_view.type == self._schema_view.storage_type:
+            return self.storage_buffer_format
```

Comment:
Can we add `return None` to be explicit? In Python, a function without `return` is commonly perceived similar to a void function in C/C++.